cluster/juju/layers/kubernetes-e2e/layer.yaml

```diff
@@ -8,5 +8,3 @@ includes:
 options:
   tls-client:
     ca_certificate_path: '/srv/kubernetes/ca.crt'
-    client_certificate_path: '/srv/kubernetes/client.crt'
```

Comment:
why are we killing these options? Not against it, just curious the reasoning.

Reply:
Because the client paths only apply to the global, shared client certificate, and both the server and client paths should be specified when the certs are explicitly requested, in case the charm uses multiple server or client certs.